### PR TITLE
Release/5.2.0

### PR DESCRIPTION
**Pull request by defunctl**
- [ADDED]: `so share <custom-wp-content-directory>` to support projects where the `wp-content` directory is named something non-standard, e.g. `content`.
- [ADDED]: `so test`: Support a custom path to a codeception `tests` directory if that is not in a standard location.

### CODE REVIEW

File: app/Commands/LocalDocker/Share.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ class Share extends BaseLocalDocker {
      *
      * @var string
      */
-    protected $signature = 'share';
+    protected $signature = 'share {directory=wp-content : The name of the wp-content folder, if renamed}';
```
Comment (nickpelton) [inline app/Commands/LocalDocker/Share.php:25]: Should this be a named argument? Like `so share --content-dir={my content/directory/path}` JIC you add other arguments later?

**Review by nickpelton — APPROVED** [review]
Looks good, one question for debate.

File: app/Commands/LocalDocker/Share.php (hunk 1 above)
Comment (defunctl) [reply app/Commands/LocalDocker/Share.php:25]: You're right, I'll update.

Comment (defunctl) [reply app/Commands/LocalDocker/Share.php]: @nickpelton updated, please have a look.

**Review by nickpelton — APPROVED** [review]

**Pull request merged**